packages/react-native-platform-override/src/Cli.ts

```diff
@@ -68,6 +68,22 @@ void doMain(async () => {
         // eslint-disable-next-line @typescript-eslint/no-misused-promises
         cmdArgv => removeOverride(cmdArgv.override!),
       )
+      .command(
+        'diff <override>',
```

Comment:
should this have `[--useManifestVersion]` at the end?

Reply:
My understanding is that yargs expects the command name + positional arguments for this